ci-operator/config/kubevirt/hyperconverged-cluster-operator/kubevirt-hyperconverged-cluster-operator-main.yaml

```diff
@@ -210,24 +256,42 @@ tests:
         requests:
           cpu: 500m
           memory: 1Gi
-    workflow: optional-operators-ci-azure-sno
-- as: hco-e2e-upgrade-prev-index-sno-azure
+    workflow: optional-operators-ci-operator-sdk-azure-sno
+- as: hco-e2e-upgrade-prev-operator-sdk-sno-azure
   optional: true
   run_if_changed: ^(api/.*|assets/.*|build/.*|ci-test-files/.*|config/.*|controllers/.*|cmd/.*|deploy/crds/.*|deploy/index-image/.*|hack/.*|pkg/.*|tests/.*|Makefile|go\.mod|go\.sum)$
   steps:
     cluster_profile: azure-virtualization
     dependencies:
-      OO_INDEX: ci-index-hco-upgrade-prev-bundle
+      OO_BUNDLE: hco-bundle-reg-prev
     env:
       BASE_DOMAIN: cnv.azure.devcluster.openshift.com
-      OO_CHANNEL: 1.9.0
       OO_INSTALL_NAMESPACE: kubevirt-hyperconverged
-      OO_PACKAGE: community-kubevirt-hyperconverged
-      OO_TARGET_NAMESPACES: '!all'
     test:
-    - as: e2e-upgrade-test
-      commands: RELEASE_DELTA=2 make upgrade-test-index-image
-      from: hco-oc-bin-image
+    - as: deploy-cr
+      commands: |
+        export OUTPUT_DIR=${SHARED_DIR}
+        export CMD=oc
+        source ./hack/compare_scc.sh
+        dump_sccs_before
+        # compressing the SCC files into one tar file, as the CI does not support sub-directories in the  SHARED_DIR
+        tar -C "${SHARED_DIR}" -czvf "${SHARED_DIR}/scc.tar.gz" scc/
+        make deploy_cr
+      from: hco-oc-bin-operator-sdk-image
+      resources:
+        requests:
+          cpu: 500m
+          memory: 1Gi
+    - as: upgrade-bundle
+      commands: |
+        # un-compressing to get all the original SCC files from the previous phase
+        tar -C "${SHARED_DIR}" -xzvf "${SHARED_DIR}/scc.tar.gz"
+        INITIAL_VERSION=1.9.0 OUTPUT_DIR=${SHARED_DIR} make upgrade-test-operator-sdk
```

Comment:
Can we set INITIAL_VERSION in the env stanza? I think it will be more visible on version bumps.

Reply:
Need to check if it'll work.trying

Reply:
done